src/blocks/content-helper/variables.scss

```
@@ -0,0 +1,37 @@
+/**
+ * This is a subset of the CSS variables defined in the Parse.ly dashboard. It
+ * is sourced from 1.44/src/styles/base.scss and defines some additional
+ * variables in the end of the file.
+ */
+
+div.wp-parsely-content-helper {
```

https://stackoverflow.com/questions/5797014/why-do-browsers-match-css-selectors-from-right-to-left/5813672#5813672

```suggestion
.wp-parsely-content-helper {
```